week-1-ts-ecosystem/w01e03-object-oriented-structure/src/model/cart.model.ts

```diff
@@ -0,0 +1,12 @@
+
+export type CartType = 'BUY_NOW' | 'AUCTION' | 'FOR_FREE';
+
+export type Currency = 'PLN';
+
+export interface ICartItem {
```

Comment:
Ten `interface` jest niepotrzebny i stwarza koszty.

Jeśli masz model danych jako `class` to już jest to:  _kształt + dane_

I w tym układzie ten interface nie robi nic sensownego - powtarza zadanie klasy.